website/src/components/IndexPage/getOrganismStatistics.ts

```diff
@@ -49,7 +50,8 @@ const getTotalAndLastUpdatedAt = async (
     const client = LapisClient.createForOrganism(organism);
     return (
         await client.call('aggregated', {
-            version: 1,
+            [VERSION_STATUS_FIELD]: siloVersionStatuses.latestVersion,
```

Comment:
Maybe we should rename the function to `getTotalUnrevokedAndLastUpdatedAt`

Reply:
Personally I think this is ok as is (but I can live with the other option too)